### PR TITLE
docs: update the jira docs

### LINKED ISSUE #3874: [Doc][jira] Jira document need to be updated for v0.15
## Documentation Scope
https://devlake.apache.org/docs/next/Plugins/jira#type-mappings-setting

## Describe the Change
Update for v0.15

### PR DESCRIPTION
**Pull request by mindlesscloud**
# Summary
update jira docs for the release v0.15.0

### Does this close any open issues?
Closes https://github.com/apache/incubator-devlake/issues/3874

### CODE REVIEW

File: docs/Plugins/jira.md [hunk 1]
```diff
@@ -140,30 +133,37 @@ To collect data, select `Advanced Mode` on the `Create Pipeline Run` page and pa
...
+        "plugin": "jira",
+        "scope": [
+          {
+            "entities": [
+              "TICKET",
+              "CROSS"
+            ],
+            "options": {
+              "boardId": 8,
+              "title": "迭代开发看板（停用）"
```
Comment (Startrekzky) [inline docs/Plugins/jira.md]: Use a board with name in English?

File: docs/Plugins/jira.md [hunk 2]
```diff
@@ -140,30 +133,37 @@ To collect data, select `Advanced Mode` on the `Create Pipeline Run` page and pa
 ```json /blueprints/{blueprintId}
```
Comment (Startrekzky) [inline docs/Plugins/jira.md]: Can we move the 'Advanced Mode' part to the `Configuration` section in the same hierarchy as 'Configure via Config UI'?

File: docs/Plugins/jira.md [hunk 3]
```diff
@@ -7,129 +7,122 @@ description: >
...
+      "standardType": "Incident",
+      "statusMappings": null
+    }
   }
-]
+}'
 ```
 
-2. Create a new data connection
+### Create a new scope
```
Comment (Startrekzky) [inline docs/Plugins/jira.md]: What kind of users will use this API? If users rarely use this, can we remove it?

**Review by Startrekzky — CHANGES_REQUESTED** [review]

File: docs/Plugins/jira.md (hunk 2 above)
Comment (mindlesscloud) [reply docs/Plugins/jira.md]: Good idea!

File: docs/Plugins/jira.md (hunk 3 above)
Comment (mindlesscloud) [reply docs/Plugins/jira.md]: you are right, I will remove them

File: docs/Plugins/jira.md (hunk 1 above)
Comment (mindlesscloud) [reply docs/Plugins/jira.md]: I will change an English name

**Review by Startrekzky — APPROVED** [review]
LGTM

**Pull request merged**